src/components/SpecialResolution/CreateSpecialResolution.vue

```diff
@@ -33,25 +42,45 @@
       />
 
       <!-- Special Resolution Form -->
-      <section id="sample-resolution-section" class="section-container mt-10">
+      <section
+        id="sample-resolution-section"
+        class="section-container mt-10"
+      >
         <header id="sample-resolution-header">
-          <h2>{{ getSpecialResolutionResource.header}}</h2>
+          <h2>{{ getSpecialResolutionResource.header }}</h2>
         </header>
 
-        <p class="section-description mt-2"
-          v-html="getSpecialResolutionResource.text"></p>
+        <p
+          class="section-description mt-2"
+          v-html="getSpecialResolutionResource.text"
+        />
 
         <div class="mt-4">
-          <v-card flat class="py-8 px-6">
+          <v-card
+            flat
+            class="py-8 px-6"
+          >
             <div class="d-flex flex-column flex-sm-row justify-center align-center">
-              <img src="@/assets/images/BCRegistries_CoopSpecialResolution-x2.png"
+              <img
+                slot-scope=""
```

Comment:
This is also a build/lint warning. We can fix this later.

![image](https://user-images.githubusercontent.com/10002889/234070058-48515a6e-4124-4cd0-be16-60265243236f.png)